lib/statsd.js

```diff
@@ -227,11 +242,53 @@ Client.prototype.send = function (stat, value, type, sampleRate, tags, callback)
 };
 
 /**
+ *
+ * @param message {String}
+ */
+Client.prototype.enqueue = function(message){
+  this.buffer += message + "\n";
```

Comment:
Wouldn't it be better to have an array and loop on each entry on flush? I don't really the like the string thing with new lines... here's an example with a PHP implementation: https://github.com/beberlei/metrics/blob/master/src/Beberlei/Metrics/Collector/StatsD.php#L95-L107